javatests/arcs/core/storage/handle/DifferentHandleManagerDifferentStoresTest.kt

```diff
@@ -0,0 +1,24 @@
+package arcs.core.storage.handle
+
+import arcs.jvm.util.testutil.TimeImpl
+import org.junit.Before
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@Suppress("EXPERIMENTAL_API_USAGE")
+@RunWith(JUnit4::class)
+class DifferentHandleManagerDifferentStoresTest : HandleManagerTestBase() {
+
+    @Before
+    fun setUp() {
+        readHandleManager = HandleManager(TimeImpl(), Stores())
+        writeHandleManager = HandleManager(TimeImpl(), Stores())
+    }
+
+    //TODO - Fix these tests for the case where there are two handle managers
+    // and they are using different store instances.
+    override fun collection_referenceLiveness() {}
+    override fun singleton_referenceLiveness() {}
+    override fun singleton_writeAndOnUpdate() {}
```

Comment:
@cromwellian Using your amazing suggestion, I was able to quickly include tests for different handle managers using different stores. These three tests are failing in that case.... to be investigated!